roles/pul_nomad/files/auto_update_override.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStartPost=
```

Comment:
Do we want this key in here twice? What does that do?

Reply:
I have no clue, that's just what was in the issue I linked to and it worked